client/src/components/Button/index.tsx

```diff
@@ -0,0 +1,121 @@
+/**  @jsxImportSource @emotion/react */
+"use client";
+
+import { css } from "@emotion/react";
+import { Button as MuiButton } from "@mui/material";
+import typography from "@/styles/typography";
+
+type ButtonProps = Parameters<typeof MuiButton>[0];
+
+function black(props: ButtonProps) {
+  return (
+    <MuiButton
+      /** MUI의 기본 Ripple Effect 비활성화 시켰습니다. 활성화할지 논의 필요합니다. */
+      disableRipple
+      css={css`
+        background-color: #343434;
+        color: #f6f6f6;
+        border-radius: 34px;
+        height: 68px;
+        padding: 16px 32px;
+        box-shadow: 9px 9px 9px rgba(0, 0, 0, 0.2);
+        ${typography.h5}
+        &:hover {
+          background-color: #7c7c7c;
+        }
+        &:active {
+          background-color: #343434;
+          border: 2px solid #f1f1f1;
+        }
+      `}
+      {...props}
+    >
+      {props.children}
+    </MuiButton>
+  );
+}
+
+function white(props: ButtonProps) {
+  return (
+    <MuiButton
+      disableRipple
+      css={css`
+        background-color: #f1f1f1;
+        color: #202020;
+        border-radius: 34px;
+        height: 68px;
+        padding: 16px 32px;
+        box-shadow: 9px 9px 9px rgba(0, 0, 0, 0.2);
```

Comment:
```suggestion
        box-shadow: 9px 9px 9px rgba(0, 0, 0, 0.2);
        border: 2px solid transparent;
```

active 일 때만 border가 생기다보니 버튼의 크기가 조금 커지면서 위치가 움직이는 문제가 발생합니다. 이렇게 같은 크기의 투명한 border를 추가하면 이를 방지할 수 있습니다!